addon/components/one-way-select.js

```diff
@@ -165,7 +165,8 @@ const OneWaySelectComponent = Component.extend(DynamicAttributeBindings, {
       foundOption = options.find(findOption);
     }
 
-    if (optionTargetPath) {
+    if (optionTargetPath &&
+      foundOption !== undefined) {
```

Comment:
Could you put the whole statement on one line?
And could you replace `foundOption !== undefined` with `isPresent(foundOption)`?